gradle.properties

```diff
@@ -10,10 +10,10 @@ sizeofVersion = 0.3.0
 jaxbVersion = 2.3.1
 
 # Terracotta clustered
-terracottaPlatformVersion = 5.8.0-pre1
-terracottaApisVersion = 1.7.0-pre1
-terracottaCoreVersion = 5.7.0-pre25
-terracottaPassthroughTestingVersion = 1.7.0-pre12
+terracottaPlatformVersion = 5.8.0-pre4
```

Comment:
@jhouserizer @chrisdennis @myronkscott @saurabhagas @akomakom : please help ⚠️ 
I am unable to release tc-platform `5.8.0-pre4`: the releaser job is failing.